src/Components/Header/header.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styles from './header.module.css'
+import { Button } from '../Button/button'
+import logo from './logo.png'
+
+export const Header = ({ projectName, isAuthenticated }) => {
+  const onLogoClick = () => {}
+  const onLoginClick = () => {}
+  const onSigninClick = () => {}
+  const onLogoutClick = () => {}
+
+  return (
+    <header className={styles.header}>
+      <div className={styles.logo} onClick={onLogoClick}>
```

Comment:
It's better to use `link` here. In case you want to use `onClick` it's better to replace `div` with `button` as better semantic alternative